src/codegen/dispatch.rs

```diff
@@ -13,21 +13,18 @@ use crate::{
 };
 use num_bigint::{BigInt, Sign};
 use num_traits::Zero;
-use solang_parser::pt;
-use solang_parser::pt::{FunctionTy, Loc};
+use solang_parser::{pt, pt::Loc};
 use std::sync::Arc;
 
 /// Create the dispatch for the Solana target
 pub(super) fn function_dispatch(
     contract_no: usize,
     all_cfg: &[ControlFlowGraph],
     ns: &mut Namespace,
+    opt: &Options,
 ) -> ControlFlowGraph {
     let mut vartab = Vartable::new(ns.next_id);
-    let mut cfg = ControlFlowGraph::new(
-        format!("dispatch_{}", ns.contracts[contract_no].name),
-        ASTFunction::None,
-    );
+    let mut cfg = ControlFlowGraph::new("solang_dispatch".into(), ASTFunction::None);
```

Comment:
Why are you removing the contract's name from the CFG name?

Reply:
emit no longer emits `solang_dispatch`, it is generated in codegen now